### PR TITLE
MM-46528_Verify Invitation Flow Tracking Telemetry 

### PR DESCRIPTION
**Pull request by julmondragon**
#### Summary
We want to track the number of invited users by admins and by end users, as well as the success rate of invitations for each.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46528

#### Related Pull Requests
- Has server changes ([please link here](https://github.com/mattermost/mattermost-server/pull/21227))

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by julmondragon** [comment]
> Mattermost test server created! 🎉
> 
> Access here: https://mattermost-webapp-pr-11286.test.mattermost.cloud
> 
> Account Type	Username	Password
> Admin	sysadmin	Sys@dmin123
> User	user-1	User-1@123

@dontoisme @sri-byte @marianunez  this test env will allow you to check the event tracking for both invite flows (Email, Copy link) by posting in the browser's console the tracking data:

Screen Shot 2022-10-05 at 1 59 05 PM

File: actions/telemetry_actions.jsx [hunk 1]
```diff
@@ -40,9 +40,11 @@ export function trackEvent(category, event, props) {
     }
 
     Client4.trackEvent(category, event, props);
-    if (isDevMode() && category === 'performance' && props) {
+
+    // !!! Only for testing purposes in cloud test env - Will be set back before merge !!!
```
Comment (pvev) [inline actions/telemetry_actions.jsx:44]: 👍

**Review by pvev — APPROVED** [review]
LGTM! thanks @julmondragon :shipit:

**Review by sri-byte — APPROVED** [review]

**Comment by stevemudie** [comment]
We'll test this post merge

**Review by dontoisme — APPROVED** [review]
Looks good to me :D

**Review by stevemudie — APPROVED** [review]

**Comment by salmanmanekia** [comment]
Sorry @marianunez & @julmondragon. I somehow skipped doing the review for this. LGTM!

**Pull request merged**